test/rich-text/commands/index.test.ts

```diff
@@ -390,6 +392,126 @@ describe("commands", () => {
         });
     });
 
+    describe("toggleTagLinkCommand", () => {
```

Comment:
I don't see any tests for un-toggling the taglink or any tests for the parameters you can pass into it. Additionally, we might want to do some active/validity checking when trying to insert taglinks into nodes where they aren't allowed.